src/components/StafferPage/StafferForm.jsx

```diff
@@ -107,10 +93,11 @@ const BaseStafferForm = ({
           required
         />
         <Field
-          name="position.organization_id"
-          component={RenderSelectField}
-          options={extractOrgChoices(stafferOptions)}
+          name="position.organization_override"
+          component={RenderInputTextField}
+          type="text"
           label={<FieldLabel text="Organization" />}
+          extraInput={{value: position.organization_override || position.organization_name}}
```

Comment:
Interesting - I might have expected the serializer to abstract this detail away for us, but alright.

Which is a better philosophical position? That the serializer should hide these details or expose them?

Reply:
You were write. I thought organization_name was coming from the Foreign key, but it does take into account org_override as well.